### DISCUSSION TITLE
Creating dataset - File and Folder structure for multi speaker datasets

### DISCUSSION THREAD

**Discussion by Vidyut** [coqui-ai/TTS#2419] [General Q&A] [answered]
I was wondering what the file structure should be or inserting speaker information, in particular.

Right now I have 
`something.wav|Text from the file`
Where does the speaker go in this?
`something.wav|Text from the file|speaker`
or
`something.wav|speaker|Text from the file`
Something else?
Also, should the audio be in the same folder or separated by speaker?
Is there any other information I can include? I'm trying to train a dataset on specific voices for an audiobook. (also does a smaller dataset mean faster training/inference? What is too small?)
Please point me to the right place if I've missed something obvious. Is there anywhere to see this information? Does it differ by datasets?

**Comment by p0p4k** [comment, marked as answer]
Hello, whatever format you want to use, you can make a function in `formatters.py` (under the tts/datasets directory). Refer to that file to understand the format.
